source/administration/incident-response-application.rst

```diff
@@ -0,0 +1,667 @@
+Incident Response Application (EE, Closed Alpha)
+------------------------------------------------------
+
```

Comment:
```suggestion
*Available in Enterprise Edition E10 & E20*

```

Reply:
@amyblais I purposely left it undefined, because packaging is in discussion for Beta and GA. I don't want to note the actual Enterprise Edition yet